web-admin/src/routes/[organization]/[project]/-/invite/+page.svelte

```diff
@@ -1,59 +1,97 @@
 <script lang="ts">
   import { goto } from "$app/navigation";
   import { page } from "$app/stores";
-  import { createAdminServiceCreateProjectWhitelistedDomain } from "@rilldata/web-admin/client";
-  import { getUserDomain } from "@rilldata/web-admin/features/projects/user-invite/selectors";
+  import {
+    createAdminServiceCreateProjectWhitelistedDomain,
+    type RpcStatus,
+  } from "@rilldata/web-admin/client";
+  import CopyInviteLinkButton from "@rilldata/web-admin/features/projects/user-invite/CopyInviteLinkButton.svelte";
+  import {
+    getUserDomain,
+    userDomainIsPublic,
+  } from "@rilldata/web-admin/features/projects/user-invite/selectors";
   import UserInviteForm from "@rilldata/web-admin/features/projects/user-invite/UserInviteForm.svelte";
   import { Button } from "@rilldata/web-common/components/button";
   import Switch from "@rilldata/web-common/components/forms/Switch.svelte";
   import Label from "@rilldata/web-common/components/forms/Label.svelte";
+  import { eventBus } from "@rilldata/web-common/lib/event-bus/event-bus";
+  import type { AxiosError } from "axios";
 
   $: organization = $page.params.organization;
   $: project = $page.params.project;
 
   let allowDomain = false;
   $: userDomain = getUserDomain();
+  $: isPublicDomain = userDomainIsPublic();
   const addToAllowlist = createAdminServiceCreateProjectWhitelistedDomain();
   async function onContinue() {
     if (allowDomain) {
-      await $addToAllowlist.mutateAsync({
-        organization,
-        project,
-        data: {
-          domain: $userDomain.data,
-          role: "viewer",
-        },
-      });
+      try {
+        await $addToAllowlist.mutateAsync({
+          organization,
+          project,
+          data: {
+            domain: $userDomain.data,
+            role: "viewer",
+          },
+        });
+      } catch (e) {
+        eventBus.emit("notification", {
+          type: "error",
+          message:
+            (e as AxiosError<RpcStatus>).response.data.message ?? e.message,
+          options: {
+            persisted: true,
+          },
+        });
+      }
     }
     return goto(`/${organization}/${project}/-/status`);
```

Comment:
We should only continue to the `/status` page if there's no error. Rather than show an ephemeral notification, we should show the error in red text below the "Continue" button.

Reply:
I disagree, error while adding a domain whitelist should not block the user from going forward. This can barely happen and there are no mocks for this. Lets do a follow up for this.

Reply:
Sounds good, I won't block on this. @ericokuma for resolution.